### ISSUE TITLE
Onion response 400(Bad request)

### ISSUE THREAD

**Issue by make-as-easy** [oxen-io/session-desktop#1894]
Hi, oxen team,
     I'm working session desktop I have 13 service nodes(oxen storage server) but facing this error 
 ```
Must have at least 2 good onion paths, actual: 1, attempt #7300 fetching more...
```
Sometimes onion path works but getting a response as BAD_REQUEST
   Last month I set up the same that time all are working fine and at that time my machines are crashed then I try to set up same but above errors are occurs.
   In the storage server, we getting errors like this
```
Could not finalize decryption
```
Last time its was working fine for me. But now it's not working like that I'm getting errors with the same setup. It seems `session desktop` was good, I thought it was coming from `storage server`

**Comment by ssuriyayuvan** [comment]
Im also facing the same issue. Please resolve this asap.

**Comment by eyeberic** [comment]
Duplicate of #1886

**Comment by Bilb** [comment]
Should be fixed in 1.7.1

**Comment by lifwon** [comment]
It's not working, the same error occurs.
